service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -1081,37 +1085,51 @@ public Boolean status() {
   }
 
   /**
-   * Builds a policy list with a single provided owner and empty reader, writer and application
-   * policies.
+   * Builds a policy list with a single provided owner and empty reader, writer, project-owner, and
+   * application policies.
    *
    * <p>This is a helper function for building the policy section of a request to create a workspace
    * resource in Sam. The provided user is granted the OWNER role and empty policies for reader,
-   * writer, and application are also included.
+   * writer, project-owner, and application are also included.
    *
    * <p>The empty policies are included because Sam requires all policies on a workspace to be
    * provided at creation time. Although policy membership can be modified later, policy creation
    * must happen at the same time as workspace resource creation.
    */
-  private Map<String, AccessPolicyMembershipV2> defaultWorkspacePolicies(String ownerEmail) {
-    Map<String, AccessPolicyMembershipV2> policyMap = new HashMap<>();
+  private Map<String, AccessPolicyMembershipRequest> defaultWorkspacePolicies(
+      String ownerEmail, String projectOwnerGroupId) {
+    Map<String, AccessPolicyMembershipRequest> policyMap = new HashMap<>();
     policyMap.put(
         WsmIamRole.OWNER.toSamRole(),
-        new AccessPolicyMembershipV2()
+        new AccessPolicyMembershipRequest()
             .addRolesItem(WsmIamRole.OWNER.toSamRole())
             .addMemberEmailsItem(ownerEmail));
+    // Optionally add a policy for project owner (used for billing projects in Terra CWB).
+    if (projectOwnerGroupId != null) {
+      policyMap.put(
+          WsmIamRole.PROJECT_OWNER.toSamRole(),
+          new AccessPolicyMembershipRequest()
+              .addRolesItem(WsmIamRole.PROJECT_OWNER.toSamRole())
+              .addMemberPoliciesItem(
+                  new PolicyIdentifiers()
+                      .resourceTypeName("billing-project")
+                      .policyName("owner")
+                      .resourceId(projectOwnerGroupId)));
+    }
     // For all non-owner/manager roles, we create empty policies which can be modified later.
     for (WsmIamRole workspaceRole : WsmIamRole.values()) {
-      if (workspaceRole != WsmIamRole.OWNER && workspaceRole != WsmIamRole.MANAGER) {
+      if (!Set.of(WsmIamRole.OWNER, WsmIamRole.MANAGER, WsmIamRole.PROJECT_OWNER)
```

Comment:
Nit (maybe for a future PR): this may be better captured in a class-level const `static final Set` rather than inline here.